LeadAlerts.Web.Tests/Controllers/HomeControllerTest.cs

```diff
@@ -12,7 +12,7 @@ public void GivenAIndexAction_ThenRendersDefaultView()
             var controller = new HomeController();
 
             controller.WithCallTo(c => c.Index())
-                .ShouldRenderDefaultView();
+                .ShouldRenderView("Index");
```

Comment:
In my head it seemed to make more sense. But as you put it that way, `ShouldRenderDefaultView` is much better. Thanks for clarifying that!